src/lib/brands.ts

```diff
@@ -0,0 +1,126 @@
+export interface BrandInfo {
+  readableName: string
+  imageName: string
+}
+
+// Copied from 'lib/device/providers.ts' in
+// 'seam-connect' on Jun 28th, 2023.
+export const brands: Record<string, BrandInfo> = {
```

Comment:
Can we pull this from the device models api instead? Basically use the data to generate this list?

Reply:
Or, actually we have `/devices/list_device_providers` and https://github.com/seamapi/javascript/blob/f1834fe551e56d0712ec36fbe2aa440cdae2c98c/src/seam-connect/routes.ts#L228 which should serve this. We may still want to hide any providers that don't have devices.

Reply:
Updated in https://github.com/seamapi/react/pull/251/commits/fa47612115b3efea1f3ecc5bc73a623a0997614a